src/config.ts

```diff
@@ -0,0 +1,58 @@
+import fs from 'fs'
+import Store from 'electron-store'
+import oldConfig from 'electron-settings'
+
+export interface LastWindowState {
+  bounds: {
+    width: number
+    height: number
+    x: number | undefined
+    y: number | undefined
+  }
+  fullscreen: boolean
+  maximized: boolean
+}
+
+const defaults = {
+  lastWindowState: ({
+    bounds: {
+      width: 800,
+      height: 600,
+      x: undefined as number | undefined,
+      y: undefined as number | undefined
+    },
+    fullscreen: false,
+    maximized: true
+  } as unknown) as LastWindowState,
+  minimalMode: false,
+  debugMode: false
+}
+
+const config = new Store({ defaults })
+
+// @TODO: Remove `electron-settings` in future version
+function migrate(): void {
+  const oldConfigFilePath = oldConfig.file()
+
+  if (!fs.existsSync(oldConfigFilePath)) {
+    return
+  }
+
+  if (oldConfig.has('debug-mode')) {
+    const debugMode = (oldConfig.get('debug-mode') as unknown) as boolean
+    config.set('debugMode', debugMode)
+    oldConfig.delete('debug-mode')
+  }
+
+  if (oldConfig.has('minimal-mode')) {
+    const minimalMode = (oldConfig.get('minimal-mode') as unknown) as boolean
+    config.set('minimalMode', minimalMode)
+    oldConfig.delete('minimal-mode')
```

Comment:
Since you are unlinking the old config, do you need to delete the individual settings as well?

Reply:
I added `fs.unlinkSync(oldConfigFile)` today, so yeah, this is not required 😅